recipes/cis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: cis
+  version: "1.3.2"
+
+source:
+  #path: ../../cis
+  git_url: https://github.com/cedadev/cis.git
+  git_tag: 1.3.2
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
+
+requirements:
+  build:
+    - python
+    - matplotlib >=1.2.0
+    - netcdf4 >=1.0
+    - numpy
+    - scipy
+    - iris >=1.8.0
+    - pyhdf  # [not win]
```

Comment:
> I could never get HDF4 to build on Windows, but I guess that's not a problem since it's already in the forge. I'll update this, thanks

I also had that problem :smile:  
That is why joining forces at conda-forge is awesome :wink: